.eslintrc.js

```diff
@@ -85,9 +85,10 @@ module.exports = {
     curly: [errorIfStrict, "multi-line"],
     eqeqeq: [errorIfStrict, "always", { null: "ignore" }],
     // Help us with emotion
-    "@emotion/syntax-preference": [errorIfStrict, "string"],
-    "@emotion/no-vanilla": errorIfStrict,
     "@emotion/import-from-emotion": ERROR,
+    "@emotion/no-vanilla": errorIfStrict,
+    "@emotion/pkg-renaming": ERROR,
```

Comment:
this flag allows us to use emotion 11 codemods (https://github.com/emotion-js/emotion/tree/master/packages/eslint-plugin#emotion-11-codemod) but I was unable to find info on the what the emotion 11 codemods are

Reply:
talked about it offline but they are basically a set of  eslint rules that help you refactor code that uses emotion when they updated to emotion 11